snaps_tests/__main__.py

```diff
@@ -47,7 +48,11 @@ def main():
 
     arguments = docopt.docopt(__doc__)
 
-    snaps_tests.config['skip-install'] = arguments['--skip-install']
+    if platform.machine() == 'armv7l':
```

Comment:
@sergiusens but project options is available only when snapcraft is running. These tests just call snapcraft. I'm not sure how you want me to access project_options.

Here I'm just checking for the machine identified as armv7l in our autopkgtests. I'm not yet preoccupied about the actual arch, because that should happen in the plugin. If it doesn't support a particular architecture, it should display a nice error to the user.